scripts/installer/glassfish-setup.sh

```diff
@@ -73,6 +73,9 @@ function preliminary_setup()
   # location of the datafiles directory: 
   # (defaults to dataverse/files in the users home directory)
   ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.directory=${FILES_DIR}"
```

Comment:
@qqmyers After some testing and looking into this with @kcondon, we are wondering - since it is now possible to configure more than one stores of the type "file", it seems like we need to be able to also define different physical directories for these locations. 
i.e., instead of the fixed `Ddataverse.files.directory=...`, it should be
`dataverse.files.file.directory=...`, `dataverse.files.file1.directory=...`, etc.
(but maybe we should also leave the current `dataverse.files.directory`, for backward compatiblity, to be consulted if `dataverse.files.file.directory` is not defined - ?)
what do you think?

Reply:
@qqmyers OK, please [half]scratch the above - I see what's going on... the code is already doing what I suggested above; it's the documentation and the migration instruction need to be updated, to include the .directory JVM option. (otherwise the file storage is working, BUT the files end up stored in /tmp/files!)

Reply:
Yep - was just writing that up since I knew I'd changed the code. I can update the docs to match.